src/clusterfuzz/_internal/base/utils.py

```diff
@@ -673,16 +675,34 @@ def restart_machine():
     os.system('sudo shutdown -r now')
 
 
-def search_bytes_in_file(search_bytes, file_handle):
+def search_bytes_in_file(search_bytes,
+                         file_handle,
+                         buffer_length=DEFAULT_SEARCH_BUFFER_LENGTH):
```

Comment:
It does not seem that anyone except tests passes a non-default buffer length. Let's remove the parameter.